### ISSUE TITLE
[Issue] Using API addShape results in error 'The executing command state is locked.'

### ISSUE THREAD

**Issue by guillaumegarcia13** [nhn/tui.image-editor#216]
## Version
Version: 3.5.3 (from the example given in the repo anyway)
You can reproduce online: https://nhn.github.io/tui.image-editor/latest/examples/example01-includeUi.html

## Development Environment
OS : Windows 10
Browser: Chrome 77.0.3835.0 (Canary), Chrome 75.0.3770.100 (Stable)

## Current Behavior
When trying to add a shape programmatically (i.e. through the provided API), there are actually 2 commands that are issued and the "Command locked' error kicks in.

![image](https://user-images.githubusercontent.com/2786038/60262295-1f7da380-98de-11e9-9f79-6da72ae44e8c.png)

Here is a very simple snippet to reproduce, simply open your DevTools and copy/paste
```js
imageEditor.addShape('circle', {
    fill          : '#ff0000',
    stroke        : '#ff0000',
    strokeWidth   : 0,
    top           : 200,
    left          : 300,
    rx            : 80,
    ry            : 80,
    isRegular     : true
})
```

## Expected Behavior
No error.

In the scenario above, no big deal but I have a more complex one where it completely messes up with the appplication.

**Comment by junghwan-park** [comment]
@guillaumegarcia13 Thank you for your report.

**Comment by guillaumegarcia13** [comment]
I guess I found a workaround.
It all boils down to the API being too entangled with the UI (activating tabs and so on...).

I managed to rewrite the strokeRange this way (I do not use it in my implementation but as I reuse part of the UI, I have to deal with event handling).
```imageEditor.ui.shape._els.strokeRange.__proto__.trigger = function() { ; };```

This is really hacky, though.

**Comment by jinwoo-kim-nhn** [comment]
Fixed in 3.9.0.
